ara/templates/navbar.html

```diff
@@ -12,16 +12,6 @@
     <div class="navbar-collapse collapse">
       <ul class="nav navbar-nav">
         <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Hosts <span class="caret"></span></a>
```

Comment:
So we'll only show playbooks then ? This is fine but I think this means we could probably do away with the navbar entirely and do a nicely featured home page like I mentioned here: https://github.com/dmsimard/ara/issues/76#issuecomment-222329270

Can be done in a follow up and I can take a stab at it.